Dockerfile

```diff
@@ -68,6 +68,8 @@ RUN mkdir /data && \
 COPY ./ /kb/module
 
 RUN mkdir -p /kb/module/work
+RUN mkdir -p /kb/module/lib
+COPY ./test/testRASTutil.pm /kb/module/lib
```

Comment:
So testRASTutil.pm should already be at /kb/module/test because of the copy on line 68 so I think you could simplify the these lines and the "use" statements in the test files. (Love that you are extracting the repetitive code though)

Reply:
Thanks. That works better than my first attempt.

Reply:
@JamesJeffryes  - I think I have made all the changes you requested.